SPECS/nodejs/nodejs.signatures.json

```diff
@@ -1,6 +1,6 @@
 {
  "Signatures": {
-  "node-v14.18.3-clean.tar.xz": "51fa0865b597e131219e6a6398961b372d773b72d28e3c75524154bd2792e0bd",
-  "clean-source-tarball.sh": "ab579872ec2f4e85a00fc1651b40a26876012256c722c3701ef3cdcb378c93d5"
+    "node-v14.20.0-clean.tar.xz": "ef614adf3a42e8aedefd374c3cd6f1510c3ee8d5cd3e6ea959763fe64c41adbb",
+    "clean-source-tarball.sh": "ab579872ec2f4e85a00fc1651b40a26876012256c722c3701ef3cdcb378c93d5"
```

Comment:
For my own learning: 
This is the first I've seen anything other than the source tarball being a component of the signatures.json file. What determines that this .sh file is also denoted and other files such as patches are not? If I ran into this same scenario when adding a new package, would I need to publish these additional files to the pipeline? Can I use the same pipeline that we use for publishing/updating tar balls?